apps/namadillo/src/App/AppSetup.tsx

```diff
@@ -57,15 +57,19 @@ export const AppSetup = ({ children }: AppSetupProps): JSX.Element => {
         containerProps={errorContainerProps}
         niceError={
           <>
-            Unable to connect to indexer. Please check your internet connection.
-            If the problem persists,{" "}
-            <button
-              className="text-yellow hover:text-cyan"
-              onClick={() => setChangeIndexerSettings(true)}
-            >
-              click here
-            </button>
-            to verify your indexer settings.
+            <p>
+              Unable to connect to indexer. Please check your internet
+              connection.
+            </p>
+            <p className="mt-1">
+              If the problem persists, you can{" "}
+              <button
+                className="text-yellow hover:text-cyan"
+                onClick={() => setChangeIndexerSettings(true)}
+              >
+                update the indexer settings.
+              </button>
+            </p>
```

Comment:
Update the copy/style of the button

![Screenshot 2024-07-11 at 16 27 17](https://github.com/user-attachments/assets/e2a5ed2f-1a6b-4778-9911-a61a1bf9c689)